### ISSUE TITLE
Crash Corrupted World.

### ISSUE THREAD

**Issue by FTBlover123** [FTBTeam/FTB-Continuum#277]
* **Modpack Version**:  1.1.0 SSP

* **Issue**: World seems to have corrupted, wasn't doing anything out of the usual.  Was just flying over to an island near to my base, so it might have something to do with some stuff unloading / becoming autoloaded (as my base was loaded via Extra Utilities) Some more notes of the initial crash: The game didn't instantly freeze, I was still able to fly around for quite a while and only realized my game had crashed when I was unable to access my bag of holding. Exploring the world I realised that no new chunks were loading and that some chunks seemed to have litterly swapped. I was able to fly around freely for about 2 minutes before the game fully froze forcing me to kill the game.

* **Link to Log or Crash File Paste**: https://paste.feed-the-beast.com/view/1af3fc61

* **Is it Repeatable?**:  No idea what caused the corruption, but the corruption seems permanent.

* **Mod/s Affected**: <! Unknown, quick glance at the crash report didn't teach anything to my untrained eye.
* **Known Fix**: 

* **Additional Comments:**: Hoping for a solution that uncorrupts the file, otherwise I'm wondering where the occasional safe of the world is kept.

**Comment by aaronhowser1** [comment]
Update to 1.3.1 and try it out then. Also, check around for backups of the world.

**Comment by FTBlover123** [comment]
Updated to 1.3.1, crashed at the same point of loading.

**Comment by FTBlover123** [comment]
Didn't mean to close.

**Comment by aaronhowser1** [comment]
Do you have any backups? I had an issue similar to this, there was a base on the server I played on that had thousands of dropped items on the floor and I had this happen whenever I got too close to it.

**Comment by FTBlover123** [comment]
Got some backups and just loaded one of them. Guess what I noticed, the island was already glitched out 2 hours before I went there (or reglitched out at my first load of the world, now in version 1.3.1)
https://i.gyazo.com/7595a9e5d3d0e192d0e5317dccf9367d.png
I remember the swapped chunks are at the same location as they were when I was able to fly around before the whole game froze. Which tells me the area must've gotten corrupted hours before.
Which is weird, as I did go to the same island earlier to grab some sand before everything crashed...

So, what to do now? I kinda feel like the island now just is a ticking timebomb for my backup...

I'm guessing this also means my old save wasn't corrupted, just that my player crashed the world every time it tried to load those specific corrupted chunks
